src/MongoDB.Driver/MapReduceOptions.cs

```diff
@@ -191,25 +205,51 @@ public static MapReduceOutputOptions Merge(string collectionName, string databas
         /// <param name="sharded">Whether the output collection should be sharded.</param>
         /// <param name="nonAtomic">Whether the server should not lock the database for the duration of the reduce.</param>
         /// <returns>A reduce map-reduce output options.</returns>
+        [Obsolete("Use a Reduce that does not have sharded and nonAtomic parameters instead.")]
```

Comment:
Slight rewording requested:

    [Obsolete("Use an overload of Reduce that does not have sharded and nonAtomic parameters instead.")]

Reply:
done